developer/src/common/delphi/compiler/CompilePackage.pas

```diff
@@ -496,6 +509,30 @@ procedure TCompilePackage.CheckKeyboardLanguages;
   begin
     if k.Languages.Count = 0 then
       WriteMessage(plsWarning, Format(SKKeyboardShouldHaveAtLeastOneLanguage, [k.ID]));
+    CheckForDuplicatedLanguages('keyboard', k.ID, k.Languages);
+  end;
+end;
+
+procedure TCompilePackage.CheckForDuplicatedLanguages(const resourceType, id: string; languages: TPackageKeyboardLanguageList);
```

Comment:
Is there a type `TPackageModelLanguageList`? (for `model.Languages` on l. 162)

Reply:
No, the name `TPackageKeyboardLanguageList` was reused for the model language list. Not ideal but this code is essentially deprecated so not planning to fix.